projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx

```diff
@@ -44,12 +45,28 @@ export class DotSelectComponent {
     _dotTouched = false;
     _dotPristine = true;
 
+    fieldAttr = {
+        type: 'dot-select',
+        name: this.name
+    };
+
     componentWillLoad() {
-        this._options = getDotOptionsFromFieldValue(this.options);
+        this.fieldAttr.name = this.name;
+        this.optionsWatch();
         this.emitInitialValue();
         this.emitStatusChange();
     }
 
+    @Watch('options')
+    optionsWatch(): void {
+        const validOptions = dotPropValidator({
+            field: { ...this.fieldAttr },
```

Comment:
Why spread?

Reply:
in here we have a couple of `attr` https://github.com/dotCMS/core-web/blob/f8c8d23f6c5bdbf043aacb9b743600769fbe898b/projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx#L48

that could endup being used on several `Watch`

Reply:
I see, but does that mean we have to spread it? why?

Reply:
to met this:
![image](https://user-images.githubusercontent.com/37185433/57794685-7306aa00-7701-11e9-9470-5157ac835d29.png)

Reply:
but why not:

```field: this.fieldAttr,```

?